aws/data_source_aws_s3_bucket_object.go

```diff
@@ -27,6 +27,10 @@ func dataSourceAwsS3BucketObject() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"bucket_key_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
```

Comment:
This attribute is read-only, so it should be marked `Computed: true` instead of `Optional: true`

```suggestion
				Computed: true,
```

Reply:
Thanks, fixed!